lib/routers/establishment/projects.js

```diff
@@ -11,13 +11,16 @@ const submit = action => (req, res, next) => {
   const params = {
     data: {
       ...req.body.data,
-      establishmentId: req.establishment.id,
-      licenceHolderId: req.user.profile.id
+      establishmentId: req.establishment.id
     },
     model: 'project',
     meta: req.body.meta || {}
   };
 
+  if (req.project) {
+    params.data.licenceHolderId = req.project.licenceHolderId;
```

Comment:
What does licenceHolderId get set to if req.project doesn't exist? Can that happen?

Reply:
that can't happen, as if req.project isn't a thing then we don't care about telling workflow who to use as the subject (new projects are insta resolved)